changelogs/unreleased/gh-6686-build-with-glibc-2-34.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/build
+
+ * Fix build errors with glibc-2.34.
```

Comment:
Nit: please append `(gh-6686)` to the message, like we usually do. Also, please link the PR and commit to the issue by adding `Closes #6686` to the commit message / PR description.